src/content/guides/getting-started.md

```diff
@@ -146,18 +149,23 @@ With that said, let's run `npx webpack` with our script as the [entry point](/co
 ``` bash
 npx webpack src/index.js --output dist/bundle.js
```

Comment:
@TheLarkInn @sokra what is the default input and output directory for webpack v4? I remember src/index.js being the default entry point if I'm not wrong. Would love to highlight the fact that you just need to run `webpack` in the terminal 🌟

Reply:
Entry https://github.com/webpack/webpack/blob/master/lib/WebpackOptionsDefaulter.js#L20

Reply:
Yeah the current setup should already be the default. I'll remove the options to highlight webpack's ease of use and default configuration ;)